src/ConfigTool.cpp

```diff
@@ -27,84 +32,182 @@ void ConfigTool::load() {
-	const String continueHtml = "<form method=\"POST\" action=\"\">";
-	const String savedAlert = "<div class=\"alert alert-success\" role=\"alert\"><button type=\"button\" class=\"close\" data-dismiss=\"alert\" aria-label=\"Close\"><span aria-hidden=\"true\">&times;</span></button>The config has been saved.</div>";
+String ConfigTool::createInput(BaseVar *item, String type) {
+	String result = "<input name=\"" + item->name + "\" type=\"" +type+ "\" value=\"";
+
+	if (!item->hideValueInWeb) {
+		result += item->toString();
+	}
+	result += "\" />";
+
+	return result;
+}
+
+String ConfigTool::createBoolSelector(ConfigVar<bool> *item, String label, String display, bool checked) {
+
+	String result = "<input name=\"" + item->name + "\" type=\"radio\" id=\"_" +label+ "_\" value=\"" +label+ "\" ";
+
+	if (checked) {
+		result += "checked ";
+	}
+	result += "><label for=\"_" +label+ "_\"> " +display+ "</label> ";
 
-	const String endHtml = "<div class=\"form-group row\"><div class=\"col-sm-1\"><button class=\"btn btn-primary\" type=\"submit\">Save</button></div><div class=\"col-sm-1 offset-sm-0\"><button type=\"button\" class=\"btn btn-danger\" onclick=\"reset()\">Reset</button></div></div></form></div></body><script>function reset(){var url=window.location.href;if(url.indexOf('?')>0){url=url.substring(0,url.indexOf('?'));}url+='?reset=true';window.location.replace(url);}</script></html>";
+	return result;
+}
+
+String ConfigTool::createWebPage(bool updated) {
+	static const String beginHtml = 
+		"<html>"
+		"<head>"
+			"<title>Config Tool</title>"
+		"</head>"
+		"<body>"
+			"<h1>ConfigTool Web Interface</h1>"
+			"<p>Edit the config variables here and click save.</p>";
+
+	static const String savedAlert = 
+			"<p>The config has been saved.</p>";
+
+	static const String continueHtml = 
+			"<form method=\"POST\" action=\"config\">"
+				"<table>";
+	
+	static const String endHtml = 
+				"</table>"
+				"<p>"
+					"<button type=\"submit\">Save</button><br/><br/>"
+					"<button type=\"submit\" name=\"reset\">Reset</button> be careful!"
+				"</p>"
+			"</form>"
+		"</body>"
+		"</html>";
 
 	String result = beginHtml;
 
 	if (updated) {
 		result += savedAlert;
 	}
-
 	result += continueHtml;
 
 	for (auto item : variables_) {
-		result += "<div class=\"form-group row\"><div class=\"col-2\"><label>" + item.first + "</label></div><div class=\"col-10\"><input name=\"" + item.first + "\" class=\"form-control\" type=\"text\" value=\"" + item.second->toString() + "\" /></div></div>";
+		
+		if (item.second->hideInWeb) continue;
+		
+		result += 
+			"<tr>"
+				"<td><label>" + item.first + "</label></td>"
+				"<td>";
+		
+		switch (item.second->varType()) {
+			
+			case VT_TEXT:
+				result += createInput(item.second, "text");
+				//<input name=\"" + item.first + "\" type=\"text\" value=\"";
```

Comment:
There is some commented out code here and at some other places. If the code won't be needed, remove it. If it could be needed in the future maybe add a comment explaining what it does and why it could be changed.